public/app/plugins/datasource/alertmanager/ConfigEditor.tsx

```diff
@@ -73,6 +73,11 @@ export const ConfigEditor = (props: Props) => {
             />
           </InlineField>
         </div>
+        {options.jsonData.handleGrafanaManagedAlerts && (
+          <span>
```

Comment:
I think we want the text to be a bit lighter, but I am not sure which class to use.

Reply:
I've sent you some pointers :)